cli/azd/pkg/tools/kubectl/models.go

```diff
@@ -107,7 +107,7 @@ type ServiceStatus struct {
 
 type Port struct {
 	Port       int    `json:"port"`
-	TargetPort int    `json:"targetPort"`
+	TargetPort any    `json:"targetPort"`
```

Comment:
nit: Can we add a comment here for future readers, to avoid a potential misunderstanding that `TargetPort` should be a strongly-typed value?